code/src/java/plugin/notes/gui/JIcon.java

```diff
@@ -54,9 +53,9 @@
 	private JPopupMenu contextMenu;
 
 	/**
-	 *  Creates new form JIcon
+	 * Creates new form JIcon
 	 *
-	 *@param  name  Name of the file to load (full path)
+	 * @param file file object for the JIcon
 	 * @param plugin
 	 */
 	JIcon(File name, NotesPlugin plugin)
```

Comment:
The parameter needs to change from name to file for the code to work. This doesn't compile.